modules/draco/test/draco-writer.spec.ts

```diff
@@ -145,8 +145,7 @@ test.skip('DracoWriter#Worker$encode(bunny.drc)', async (t) => {
   t.end();
 });
 
-// TODO v4.0 restore this test
-test.skip('DracoWriter#WorkerNodeJS#encode(bunny.drc)', async (t) => {
+test('DracoWriter#WorkerNodeJS#encode(bunny.drc)', async (t) => {
```

Comment:
Nit: Do we need special Node.js worker tests once this all works? Can't we just remove the `if (isBrowser)` from the main test?

Reply:
The previous test in this test suite is for browser. It is skipped because of "Error: Failed to execute 'importScripts' on 'WorkerGlobalScope'". See comment for that test